cividiscount.php

```diff
@@ -429,7 +429,7 @@ function cividiscount_civicrm_buildAmount($pageType, &$form, &$amounts) {
                 $recalculateTaxAmount = array();
                 if (array_key_exists('tax_amount', $originalAmounts[$fee_id]['options'][$option_id]) &&
                     array_key_exists('tax_rate', $originalAmounts[$fee_id]['options'][$option_id])) {
-                  $recalculateTaxAmount = CRM_Contribute_BAO_Contribution_Utils::calculateTaxAmount($amount, $originalAmounts[$fee_id]['options'][$option_id]['tax_rate']);
+                  $recalculateTaxAmount = CRM_Contribute_BAO_Contribution_Utils::calculateTaxAmount($amount, $originalAmounts[$fee_id]['options'][$option_id]['tax_rate'], TRUE);
```

Comment:
The last TRUE is for $ugWeDoNotKnowIfItNeedsCleaning_Help - but haven't you just ensured it is clean? You should pass something clean rather than use that because that param will be grandfathered out

Reply:
Application of changes did solve problem in my case !